docs/sessions.md

```diff
@@ -0,0 +1,19 @@
+# Sessions
+
+Altis includes support for PHP sessions using Redis as the storage backend. In order to activate PHP sessions support you need to have Redis activated as well, eg:
+
+```json
+{
+	"extra": {
+		"altis": {
+			"modules": {
+				"cloud": {
+					"redis": true
+				}
+			}
+		}
+	}
+}
+```
+
+Worth noting is that the cookie name is `altis_session`, rather than the default `PHPSESSID`.
```

Comment:
There's a general pattern in the docs like `**Note:** ...` that we could use here, for consistency.

```suggestion
**Note:** The cookie name is `altis_session`, rather than the default `PHPSESSID` in order to bypass CloudFront page caching.
```